include/swift/AST/GenericSignatureBuilder.h

```diff
@@ -826,7 +835,7 @@ class GenericSignatureBuilder::RequirementSource final
     TypeBase *type;
 
     /// A protocol conformance used to satisfy the requirement.
-    ProtocolConformance *conformance;
+    void *conformance;
```

Comment:
Why can't this just be a ProtocolConfomanceRef? You're including the header above